app/views/shared/proxy_rules/_table.html.slim

```diff
@@ -11,6 +11,12 @@ table#mapping_rules.data
       th = sortable('proxy_rules.position', 'Position')
       th.actions
         = link_to 'Add Mapping Rule', new_rule_path, class: 'action add'
+    tr.search
+      = search_form do |s|
+        th colspan="6"
```

Comment:
```suggestion
        th colspan="3"
```

No need for the search box to be this wide i think

Reply:
Also, search input should be in the second column as the first column is the verb